final/scripts/smooth_final.py

```diff
@@ -4,22 +4,20 @@
 Imports functions from multiple locations, see test files in correct folders
 
 Potential variants:
-	- this currently uses sigma =1.5/ fwhm =3.5, see lines 90-92
```

Comment:
remove the stuff about fwhm, note that fwhm = (2*np.sqrt(2 *np.log(2))) * sigma,
and the original paper used much smoother data (with fwhm as 5.5 mm)